src/helm/proxy/models.py

```diff
@@ -374,6 +374,31 @@ def engine(self) -> str:
         "JavaScript subset of The Stack (v1.1).",
         tags=[CODE_MODEL_TAG],
     ),
+    # Local HuggingFace models
+    Model(
```

Comment:
Could you amend the PR title and description to say that it also adds LLaMA and Alpaca?